docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -747,77 +1040,66 @@ We recommend you explore all Terraform files. Below is a high-level overview of
 <br /> 
 
 ### Deploy Terraform
-After you update the **terraform.tfvars** file and carefully review the other files, initialize the Terraform provider.
-<br />
+
+After updating the **terraform.tfvars** file and carefully reviewing the other files, initialize the Terraform provider.
 
 ```bash
 terraform init
 ```
 
-The `init` command downloads plugins and providers from the **provider.tf** file. Next, preview the resources Terraform will create. 
-<br />
+The `init` command downloads the plugins and providers from the **provider.tf** file. Next, use the `plan` command to preview the resources that Terraform will create. 
 
 ```bash
 terraform plan
 ```
 
-The output displays the resources Terraform will create in an actual implementation. 
-<br />
+The output displays the resources that Terraform will create in an actual implementation. 
 
 ```bash
 # Output condensed for readability
 Plan: 2 to add, 0 to change, 0 to destroy.
 ```
 
-Finish creating all the resources. 
-<br />
+Complete the creation of all the resources. 
 
 ```bash
 terraform apply -auto-approve
 ```
 
-It can take up to 20 minutes to provision the cluster. When cluster provisioning completes, the following message displays. 
-<br />
+It can take up to 20 minutes to provision the cluster. Once the cluster provisioning is complete, the following message displays. 
 
 ```bash
 # Output condensed for readability
 Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
 ```
 
-You can observe the cluster deployment progress in Palette by navigating back to Palette.
-<br /> 
+You can monitor the progress of the cluster deployment in the Palette interface.
 
 
 #### Check the In-Progress Deployment
-Log into the [Palette](https://console.spectrocloud.com/), and navigate to the **Profile** section in the left **Main Menu**. If the Terraform deployment is successful, the newly created cluster profile is displayed as shown in the screenshot below. 
-
-<br />
+Log in to [Palette](https://console.spectrocloud.com/) and navigate to the **Profile** section in the left **Main Menu**. If the Terraform deployment is successful, the newly created cluster profile is displayed as shown in the screenshot below. 
 
 ![Screenshot of the successful Profile in Palette.](/tutorials/deploy-pack/registries-and-packs_deploy-pack_verify-profile.png)
 
```

Comment:
Wouldn't it make more sense to view the cluster vs the cluster profile?  Otherwise this section is not really helpful and could be removed.

Reply:
Yes, indeed. I removed the cluster profile example and added a cluster example.